pkg/printers/internalversion/describe.go

```diff
@@ -3273,7 +3273,11 @@ func printNetworkPolicySpecIngressFrom(npirs []networking.NetworkPolicyIngressRu
 		} else {
 			for _, from := range npir.From {
 				w.Write(LEVEL_0, "%s", initialIndent)
-				if from.PodSelector != nil {
+				if from.PodSelector != nil && from.NamespaceSelector != nil {
+					w.Write(LEVEL_0, "%s: %s\n         %s: %s\n",
```

Comment:
What's the space for? Can you use`initialIndent`?

Reply:
Sorry I'm not very familiar with `initialIndent` you mentioned. I was use those space to make sure `PodSelector` and `NamespaceSelector` are aligned. Like:
```
From NamespaceSelector: id=ns1,id2=ns2
         PodSelector: id=pod1,id2=pod2
```
And
```
To NamespaceSelector: id=ns1,id2=ns2
     PodSelector: id=pod1,id2=pod2
```
Apparently `from` and `to` have different indentation.

Or, maybe I can use `w.Write(LEVEL_1...`, to make the output like:
```
From
    NamespaceSelector: id=ns1,id2=ns2
    PodSelector: id=pod1,id2=pod2
```
You think this would be better?

Reply:
`initialIndent` is the string before each line. In this case, it's `"    "` (4 spaces), see the caller of `printNetworkPolicySpecIngressFrom`. 

Here, `initialIndent` is added before `From` when printing (L3275). `initialIndent` may vary, so instead of specifying a fixed number of space here, it should be something like `initialIndent + "     "`. 

Using `LEVEL_0` is fine. Would you also separate these two lines into two `w.Write` calls, instead of putting two lines in the same `w.Write`? It'll be easier to read.